packages/orbit-components/src/Itinerary/__tests__/index.test.tsx

```diff
@@ -4,15 +4,30 @@ import { renderHook, act } from "../../test-utils";
 import { ItineraryProvider, useWidth } from "../context";
 
 describe("Itinerary context", () => {
+  it("should have minimum width of 70", () => {
+    const wrapper = ({ children }) => <ItineraryProvider>{children}</ItineraryProvider>;
+
+    const { result } = renderHook(() => useWidth(), { wrapper });
+
+    act(() => {
+      result.current.setWidth(30);
+      result.current.setWidth(50);
+      result.current.setWidth(40);
+    });
+
+    expect(result.current.calculatedWidth).toBe(70);
+  });
   it("should have calculated width", () => {
     const wrapper = ({ children }) => <ItineraryProvider>{children}</ItineraryProvider>;
 
     const { result } = renderHook(() => useWidth(), { wrapper });
 
     act(() => {
-      result.current.setWidths([30, 40, 50]);
+      result.current.setWidth(60);
```

Comment:
Are these just random numbers to check that it correctly picks the maximum?

Reply:
Yes, they're random. I just wanted to make sure one would be higher than the default value 70 in the context.

It's a bit of a magic value and I would prefer for it to be a constant (since it's also used as a default value elsewhere). But I'm not sure about the reasoning behind that value so didn't want to propose any bigger change for now.

As I mentioned below, if we agree this is a good approach, I think there would still be one more step/commit to rename things better for them to be more descriptive as to what they really do, including that magic 70 value.

Reply:
Agreed. To me they can be renamed

Reply:
Alright, added my proposal to rename them :)